books/projects/x86isa/machine/instructions/shifts-spec.lisp

```diff
@@ -692,12 +682,17 @@ set to the most-significant bit of the original operand.</p>"
         (input-rflags :type (unsigned-byte 32)))
 
        :parents (sar-spec)
-       :guard-hints (("Goal" :in-theory (e/d* (rflag-RoWs-enables)
-                                              (unsigned-byte-p
-                                               (tau-system)))))
+       :guard-hints
+       (("Goal" :in-theory (e/d* (rflag-RoWs-enables)
+                                 (unsigned-byte-p
+                                  bitops::logbit-to-logbitp
+                                  (tau-system))))
+        (and stable-under-simplificationp
+             '(:in-theory (enable bitops::logbit-to-logbitp))))
+       :guard-debug t
```

Comment:
I think that `:guard-debug t` makes things a little slower, so it would be good to remove.

Reply:
Good catch, left that in on accident. There are a handful of them throughout the code, which we should probably remove.